kivy/tests/test_widget.py

```diff
@@ -118,3 +118,58 @@ def test_export_to_png(self):
         rmtree(tmp)
 
         self.root.remove_widget(wid)
+
+    def test_unknown_property(self):
+        from kivy.uix.widget import Widget
+        with self.assertRaises(ValueError) as cm:
+            Widget(width=12, unkn="abc")
+        self.assertIn("Unexpected properties ['unkn']", str(cm.exception))
+
+    def test_known_property_multiple_inheritance(self):
+        class Event:
```

Comment:
This should just be `EventDispatcher` directly, because that's what we're testing.